packages/kolibri-tools/lib/lint.js

```diff
@@ -221,14 +220,25 @@ function lint({ file, write, encoding = 'utf-8', silent = false } = {}) {
           let block;
           // First lint the whole vue component with eslint
           formatted = eslint(source);
+
+          let vueComponent = compiler.parseComponent(formatted);
+
+          // Format template block
+          if (vueComponent.template && vueComponent.template.content) {
+            formatted = insertContent(
```

Comment:
Did you do this because you were seeing instances of the block indentation and line spacing being stripped by eslint?

Reply:
This logic was added because I noticed that auto-indentation is already working for `style` and `script` blocks => HTMLHints's `"'Top-level content should be surrounded by one empty line.'"` and `"Top-level content should be indented two spaces."` was duplicate behaviour.

=> I wanted to remove these two duplicate checks from HTMLHint but I noticed that I would still need to leave some of them there because of `template` block which indent wasn't auto-formatted. I didn't want to make both formatting process and HTMLHint responsible for basically the same thing, so instead of removing those two HTMLHint checks only partially, I added auto-indent to template block which gave me the freedom to get rid of `"'Top-level content should be surrounded by one empty line.'"` and `"Top-level content should be indented two spaces."` completely.

Reply:
There's probably no need to insert 2 spaces indent again to template block (`vue/html-indent` should take care of it), but there's still need to add an empty line to the beginning/end of the template block and for that purpose, I just re-used the logic we have already and yeah, right now it causes duplicate behavior of 2 spaces indent in template because they're inserted again in the section you referenced - not exactly harmful but also kind of useless and misleading. I suppose that adding `indent` Bool parameter to `insertContent` and setting it to `false` at this particular place could solve this for now. Is this what you mean or is there anything else?

Reply:
Yeah - maybe the best thing to do is to add this flag for insertContent, and only set it to true for style blocks. You can then add in the `vue/html-indent` and `vue/script-indent` rules to handle the other two cases.

This paves the way for some follow up work that could hopefully get rid of the need for the function all together, by adding a `vue/style-indent` rule, and a `vue/line-space-before-and-after-blocks` rule (or something snappier, maybe!).